modules/virt-binding-devices-vfio-driver.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-binding-devices-vfio-driver_{context}"]
+= Binding a PCI device to the VFIO driver
+
+To bind a PCI device to the VFIO driver, find the values for `vendor-ID` and `device-ID` and add them to the `/etc/modprobe.d/vfio.conf` file. Then, configure the `MachineConfig` object file for the VFIO driver and load the new driver.
```

Comment:
The `MachineConfig` object is spec that describes what what should be done and on which node.
In our case, we want the system to create a 2 file `/etc/modprobe.d/vfio.conf` and `/etc/modules-load.d/vfio-pci.conf` on the worker nodes (or on the nodes that has a relevant PCI device) - It is up to the administrator to create these MachineConfig objects according to the cluster layout.

File `/etc/modules-load.d/vfio-pci.conf` - will load the vfio-pci kernel module on the worker nodes - the content of this file will be `vfio-pci`
File `/etc/modprobe.d/vfio.conf` - options vfio-pci ids=[list of pci-ids separated by comma] - in our example there is only one device so it will be `options vfio-pci ids=10de:1eb8`

Perhaps, we should word this differently... something like the following?
```  
To bind the relevant PCI devices to the VFIO driver, find the values for `vendor-ID` and `device-ID` for each of them. Add the combined list of `vendor-ID:device-ID`s to the `MachineConfig` object file. The Machine Config Operator will generate a `/etc/modprobe.d/vfio.conf` file on the nodes to bind these devices to the VFIO driver.
```

Reply:
I've updated the content - please review.